packages/ui/src/composites/wui-tabs/index.ts

```diff
@@ -3,6 +3,7 @@ import { customElement, property, state } from 'lit/decorators.js'
 import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
 import type { IconType } from '../../utils/TypesUtil.js'
 import styles from './styles.js'
+import { animate } from 'motion'
```

Comment:
`motion` is not a dependency of ui package (it is originally found in scaffold). Let's move it from there to ui to be more accurate with deps.